### ISSUE TITLE
Super secret link in responsive menu

### ISSUE THREAD

**Issue by ghost** [paircolumbus/paircolumbus.github.io#50]
So if you view the Pair Columbus app on a mobile device, and push the responsive menu button, a link without any text appears in between the links to "getting started" and "people" links. The blank link leads to the CSS stylesheet of the app.

![screenshot_2016-03-05-10-13-35](https://cloud.githubusercontent.com/assets/6874476/13548452/c917b0b8-e2be-11e5-8ed8-7d98f8841666.png)
![screenshot_2016-03-05-10-13-42](https://cloud.githubusercontent.com/assets/6874476/13548454/d61fadec-e2be-11e5-859a-89823d6ad133.png)

**Comment by nickserv** [comment]
Good catch. This happens because SCSS files are used as templates, in Jekyll, so it thinks it's a page, so it adds it to the navbar. We should ensure navbar links only include files that we want users to load directly.
